tests/smoke-tests/defaults.test.ts

```diff
@@ -94,43 +103,56 @@ for (let packageManager of SUPPORTED_PACKAGE_MANAGERS) {
       expect(exitCode).toEqual(0);
     });
 
-    it('build and test', async () => {
-      let addonFixture = fixturify.readSync('./fixtures/addon');
-      fixturify.writeSync(join(addonDir, 'src'), addonFixture);
-
-      let testFixture = fixturify.readSync('./fixtures/rendering-tests');
-      fixturify.writeSync(join(addonDir, 'tests/rendering'), testFixture);
-
-      // Ensure that we have no lint errors.
-      // It's important to keep this along with the tests,
-      // so that we can have confidence that the lints aren't destructively changing
-      // the files in a way that would break consumers
+    it('lint:fix with no fixtures', async () => {
       let { exitCode } = await execa({ cwd: addonDir })`${packageManager} run lint:fix`;
 
       expect(exitCode).toEqual(0);
+    });
 
-      let buildResult = await execa({ cwd: addonDir })`${packageManager} run build`;
+    describe('with fixture', () => {
+      beforeEach(async () => {
+        let addonFixture = fixturify.readSync('./fixtures/addon');
+        fixturify.writeSync(join(addonDir, 'src'), addonFixture);
 
-      expect(buildResult.exitCode).toEqual(0);
+        let testFixture = fixturify.readSync('./fixtures/rendering-tests');
+        fixturify.writeSync(join(addonDir, 'tests/rendering'), testFixture);
+      });
 
-      let contents = await dirContents(join(addonDir, 'dist'));
+      it('lint:fix', async () => {
+        let { exitCode } = await execa({ cwd: addonDir })`${packageManager} run lint:fix`;
+
+        expect(exitCode).toEqual(0);
+      });
+
+      it('build', async () => {
+        let buildResult = await execa({ cwd: addonDir })`${packageManager} run build`;
+
+        expect(buildResult.exitCode).toEqual(0);
+
+        let contents = await dirContents(join(addonDir, 'dist'));
+
+        expect(contents).to.deep.equal(['_app_', 'components', 'index.js', 'index.js.map']);
+      });
 
-      expect(contents).to.deep.equal(['_app_', 'components', 'index.js', 'index.js.map']);
+      it('test', async () => {
+        // It's important that we ensure that dist directory is empty for this test, because
+        await fs.rm(join(addonDir, 'dist'), { recursive: true, force: true });
```

Comment:
important -- we have to delete the `dist` directory before we run tests otherwise we can have false positives